src/main/scala/nl.knaw.dans.easy.bag2deposit/DepositPropertiesFactory.scala

```diff
@@ -56,13 +60,18 @@ case class DepositPropertiesFactory(configuration: Configuration) {
       addProperty("identifier.urn", urn)
       addProperty("identifier.fedora", fedoraId)
       addProperty("dataverse.bag-id", "urn:uuid:" + bagInfo.uuid)
-      addProperty("dataverse.sword-token", "urn:uuid:" + bagInfo.versionOf.getOrElse(bagInfo.uuid))
-      addProperty("dataverse.nbn", "urn:uuid:" + bagInfo.versionOf.map(getBaseUrn).getOrElse(urn))
+      addProperty("dataverse.sword-token", bagInfo.versionOf.getOrElse(bagInfo.uuid))
+      addProperty("dataverse.nbn", bagInfo.versionOf.map(getBaseUrn).getOrElse(urn))
       if (!configuration.dansDoiPrefixes.contains(doi.replaceAll("/.*", "/")))
-        addProperty("dataverse.other-id", doi)
+        addProperty("dataverse.other-id", "https://doi.org/"+doi)
```

Comment:
this assumes the `other-id` is a doi. maybe we need to be more careful with this in the future

Reply:
The `doi` in this context is actually the first `<dcterms:identifier xsi:type="id-type:DOI">`